e2e/iam/projects_test.go

```diff
@@ -47,6 +47,7 @@ func TestProjects(t *testing.T) {
 			projectsEntity,
 			"create",
 			projectName,
+			"--withoutDefaultAlertSettings",
```

Comment:
[nit] can you please create a different scenario, ie a test with the flag and a test without the flag set? extra points and a verification that when using the flag the alerts are not present

Reply:
I was actually going to remove this flag from the e2e tests as the ops manager iam test would fail because of it. I don't think is worth creating a new project_iam_e2e file only for testing one flag but let me know what you think.

Reply:
another option would be to check the service in the help file but it seems a lot of work for just one flag.